shared-bindings/supervisor/Runtime.c

```diff
@@ -52,6 +55,21 @@ STATIC supervisor_run_reason_t _run_reason;
 //|         ...
 //|
 
+//|     usb_connected: bool
+//|     """Returns the USB mount status (read-only)."""
```

Comment:
```suggestion
//|     """Returns the USB enumeration status (read-only)."""
```

Let's not say `mount` because it doesn't have to do with the filesystem (even though TinyUSB uses this term.)